website/docs/feature-flag-tutorials/golang/golang-examples.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Go feature flag examples
+slug: /feature-flag-tutorials/golang/examples
+---
+
+In our [Go feature flag tutorial](/feature-flag-tutorials/go), we implemented a simple feature flag that could be turned on and off. This document will walk you through other examples of what can be achieved using feature flags in Go.
```

Comment:
That's exactly it, this was mostly for reference purposes. So that people have easy blocks of code to copy-paste for various actions in their favourite languages. We've done this for every language we support so far.